doc/classes/Callable.xml

```diff
@@ -46,6 +46,24 @@
 		    # Prints "Attack!", when the button_pressed signal is emitted.
 		    button_pressed.connect(func(): print("Attack!"))
 		[/codeblock]
+		In GDScript, you can access methods and global functions as [Callable]s using property syntax.
+		[codeblock]
+		func _init():
+		    create_tween().tween_callback(node.queue_free)  # Object methods.
+		    create_tween().tween_callback(array.clear)  # Methods of built-in types.
+		    create_tween().tween_callback(print.bind("Test"))  # Global functions.
+		[/codeblock]
+		[b]Note:[/b] [Dictionary] doesn't support above due to ambiguity with keys.
+		[codeblock]
+		func _init():
+		    var dictionary = {"hello": "world"}
```

Comment:
```suggestion
		    var dictionary = { "hello": "world" }
```

Reply:
I think this is not a recommended style, actually.

Reply:
Ah, hm. It is not actually specified in the style guide at all.

Reply:
Yes, but it's presented [here](https://docs.godotengine.org/en/stable/tutorials/scripting/gdscript/gdscript_basics.html):

![](https://github.com/godotengine/godot/assets/47700418/e4486382-23ca-40e7-93a0-230d9ce80bd2)